UPGRADE-1.12.md

```diff
@@ -115,3 +111,29 @@ Therefore you need to update your code to follow this change.
    + "rollup-plugin-uglify": "^6.0.2",
    ```
     Please visit [3.x Migration Guide](https://www.chartjs.org/docs/latest/getting-started/v3-migration.html) for more information.
+
+Example changes we made in our codebase to adjust it to the new version of dependencies might be found [here](https://github.com/Sylius/Sylius/pull/14319/files).
+Remember, when you face any issues while updating your dependencies you might ask for help in our [Slack](https://sylius-devs.slack.com/) channel.
+
+#### Webpack becomes our default build tool 🎉
+
+`1.12` comes with a long-awaited change - Webpack becomes our default build tool. 
```

Comment:
Even though I share the enthusiasm 😄 I would keep upgrade files as _rough_ as possible (purely to make it only informative) and keep the emotions for blogposts or tweets 🚀

Reply:
GitHub doesn't show this is outdated but I've changed it 💃🏻